docs/general/builtins/scalar.rst

```diff
@@ -2669,6 +2669,32 @@ Example:
     +------------------------+
     SELECT 1 row in set (... sec)
 
+.. _pg_get_function_result:
+
+``pg_get_function_result``
+--------------------------
+
+The function ``pg_get_function_result`` returns the text representation of the
+return value's data type of the function referred to by the OID.
```

Comment:
`return value's data type of the function referred **to** by the OID.` is **to** obsolete here?